src/operator/tensor/elemwise_unary_op.cc

```diff
@@ -454,9 +454,9 @@ MXNET_OPERATOR_REGISTER_BINARY(_backward_radians)
 
 // sinh
 MXNET_OPERATOR_REGISTER_UNARY(sinh)
-.describe(R"code(Hyperbolic sine, element-wise.
+.describe(R"code(Returns element-wise Hyperbolic sine value of the input.
```

Comment:
hyperbolic should be lowercased. And don't say "sine value". 

Just:

"Returns the element-wise hyperbolic sine of the input array, computed element-wise."